app/src/main/java/com/dessalines/thumbkey/keyboards/ENMessageEaseSymbols.kt

```diff
@@ -890,14 +890,395 @@ val KB_EN_MESSAGEEASE_SYMBOLS_SHIFTED =
         ),
     )
 
+val KB_EN_MESSAGEEASE_SYMBOLS_NUMBERS =
```

Comment:
This just got added at https://github.com/dessalines/thumb-key/blob/main/app/src/main/java/com/dessalines/thumbkey/keyboards/NumericMessagEase.kt . So use that common one.

Reply:
Note that that particular one is not the english symbol layout, so I will have to tweak some of the positions slightly